config/locales/en.yml

```diff
@@ -30,4 +30,19 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  views:
+    pagination:
+      first: "&laquo; First"
+      last: "Last &raquo;"
+      previous: "&lsaquo; Prev"
+      next: "Next &rsaquo;"
+      truncate: "&hellip;"
+  helpers:
+    page_entries_info:
+      one_page:
+        display_entries:
+          zero: "No %{entry_name} found"
+          one: "Showing <b>1</b> %{entry_name}"
```

Comment:
the `one:` is used, but the `zero:` is currently handled by your zero search results PR. I guess I thought it was belt and braces, but I will remove it.